superset-frontend/src/components/FilterableTable/FilterableTable.tsx

```diff
@@ -555,6 +560,7 @@ export default class FilterableTable extends PureComponent<
         style={{ height }}
         className="filterable-table-container"
         ref={this.container}
+        css={filterableTableStyles}
```

Comment:
the css prop approach works, but I kind of lean toward the `styled.div` implementation like you did with `StyledPane`. If you did that, you get the bonus win of being able to kill this off from `superset-frontend/src/SqlLab/main.less`:
```
.filterable-table-container {
  margin-top: 48px;
}
```
... and switch it to `margin-top: ${({ theme }) => theme.gridUnit * 12}px;` for a bonus bonus win ;)

Reply:
ok, great. I didn't see that style in less, that's another win for sure. I was thinking something like if you can utilize the style, use styled.div, etc, otherwise css prop?